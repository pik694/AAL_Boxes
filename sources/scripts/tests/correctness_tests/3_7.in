3
75 50 73
46 56 33
94 5 79

