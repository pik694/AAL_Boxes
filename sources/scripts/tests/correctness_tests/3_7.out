2 1 0 
395888->310880
